### PR TITLE
Make confirmation and certificate email optional

### PR DESCRIPTION
**Pull request by danypr92**
Create three boolean fields in Company to choose if we want to send the confirmation, capital release and certificate emails in SubscriptionRequest flow.

### CODE REVIEW

**Comment by enricostano** [comment]
@houssine78 @robinkeunen we would like to propose a way to opt out email sending for "confirmation" and "certificate" templates with a configuration at company level.

What do you think?

Also, we would like to add tests to this PR but we're not sure what's the best way to do that. Any hint?

Thanks!

File: easy_my_coop/models/coop.py [hunk 1]
```diff
@@ -89,8 +89,9 @@ def create(self, vals):
             cooperator.write({"cooperator": True})
         subscr_request = super(SubscriptionRequest, self).create(vals)
 
-        mail_template_notif = subscr_request.get_mail_template_notif(False)
-        mail_template_notif.send_mail(subscr_request.id)
+        if self._send_confirmation_email():
+            mail_template_notif = subscr_request.get_mail_template_notif(False)
```
Comment (robinkeunen) [inline easy_my_coop/models/coop.py]: We should name the argument for clarity here and later (I know it's not your code :p):

```py
get_mail_template_notif(is_company=False)
```

File: easy_my_coop/models/company.py [hunk 2]
```diff
@@ -92,6 +92,9 @@ def _compute_base_logo(self):
         translate=True,
         help="Text to display aside the checkbox to approve financial risk.",
     )
+    send_certificate_email = fields.Boolean(string="Send certificate email", default=True)
+    send_confirmation_email = fields.Boolean(string="Send confirmation email", default=True)
+    send_capital_release_email = fields.Boolean(string="Send Capital Release email", default=True)
```
Comment (robinkeunen) [inline easy_my_coop/models/company.py]: Can you add in roadmap.rst "gather and consolidate all easy_my_coop settings in  the application parameters"?

**Review by robinkeunen — CHANGES_REQUESTED** [review]
Minor changes asked, otherwise good for me. 👍 


I also asked a review from @houssine78

File: easy_my_coop/models/company.py (hunk 2 above)
Comment (houssine78) [inline easy_my_coop/models/company.py]: could you format the code to comply with the actual formating ?

File: easy_my_coop/models/coop.py (hunk 1 above)
Comment (houssine78) [reply easy_my_coop/models/coop.py]: +1

**Review by houssine78 — APPROVED** [review]

File: easy_my_coop/models/company.py (hunk 2 above)
Comment (danypr92) [reply easy_my_coop/models/company.py]: Sorry @robinkeunen but I can't find the file `roadmap.rst` in this module. Can you please point me to the file?

I only find a `ROADMAP.rst` file in the `easy_my_coop_api` module.

Comment (danypr92) [reply easy_my_coop/models/company.py]: Of course! Done!

File: easy_my_coop/models/coop.py (hunk 1 above)
Comment (danypr92) [reply easy_my_coop/models/coop.py]: Done!

I change all the calls to the `get_mail_template_notif` method to notify the argument names.

File: easy_my_coop/models/company.py (hunk 2 above)
Comment (robinkeunen) [reply easy_my_coop/models/company.py]: You can create the file in the `readme` directory :-)

**Review by robinkeunen — APPROVED** [review]

File: easy_my_coop/models/company.py (hunk 2 above)
Comment (danypr92) [reply easy_my_coop/models/company.py]: Done! :man_dancing:

**Comment by enricostano** [comment]
@houssine78 @robinkeunen I think all the comments have been addressed. Can this be merged? Thanks!

File: easy_my_coop/models/company.py (hunk 2 above)
Comment (houssine78) [reply easy_my_coop/models/company.py]: thank you

**Comment by houssine78** [comment]
it's a go for me ;) big thanks for your contribution @danypr92 and @enricostano

**Pull request merged**